yang/IETF-04/ietf-l3vpn-ntw.yang

```diff
@@ -1406,6 +1406,74 @@ module ietf-l3vpn-ntw {
                            diagnosis purposes. The semantic of the description
                            is local to an implementation.";
                       }
+                      leaf as-override {
+                        type boolean;
+                          default false;
+                          description
+                           "Define whether AS override is enabled, i.e., replace
+                           the AS number of the peer specified in the AS Path
+                           attribute with the local AS number.";
+                        }
+
+                      leaf default-route {
+                        type boolean;
+                        default false;
+                        description
+                         "Define whether default route can be advertised
+                         to its peer.";
+                        }
+
+
+                      container bgp-max-prefix {
+                        leaf max-prefix {
+                          type uint32;
+                          default 5000;
+                        }
+
+                      leaf warning-threshold{
+                        type decimal64 {
+                          fraction-digits 5;
+                          range "0..100";
+                        }
+                          units percent;
+                        default 75;
+                        }
+
+                      leaf violate-action {
```

Comment:
add reference  warning-threshold 
Add comment that the behaviour is to be passed to the BGP configuration